kratos/python/add_kernel_to_python.cpp

```
@@ -42,6 +42,19 @@ const TVariableType& GetVariable(
     return TVariableType::StaticObject();
 }
 
+bool HasConstitutiveLaw(Kernel& rKernel, const std::string& constitutive_law_name) {
+    return KratosComponents<ConstitutiveLaw>::Has(constitutive_law_name);
+}
+
+const ConstitutiveLaw& GetConstitutiveLaw(
+    Kernel& rKernel, const std::string& constitutive_law_name) {
+    if (KratosComponents<ConstitutiveLaw>::Has(constitutive_law_name)) {
+        return KratosComponents<ConstitutiveLaw>::Get(constitutive_law_name);
+    }
+
```

I think you should throw an exception

---

done

---

i am being very picky here, but it would be cool to print the list of available CLs

---

this is how it looks like now:
```
RuntimeError: Error: The requested Constitutive Law "LinearElastic355DLaw" is unknown!
Maybe you need to import the application where it is defined?
The following Constitutive Laws are available:
	BeamConstitutiveLaw
	ConstitutiveLaw
	HyperElastic3DLaw
	HyperElasticPlaneStrain2DLaw
	KirchhoffSaintVenant3DLaw
	KirchhoffSaintVenantPlaneStrain2DLaw
	KirchhoffSaintVenantPlaneStress2DLaw
	LinearElastic3DLaw
	LinearElasticAxisym2DLaw
	LinearElasticOrthotropic2DLaw
	LinearElasticPlaneStrain2DLaw
	LinearElasticPlaneStress2DLaw
	TrussConstitutiveLaw

```